rmw_connext_shared_cpp/src/node.cpp

```diff
@@ -58,6 +59,18 @@ create_node(
     RMW_SET_ERROR_MSG("failed to get default participant qos");
     return NULL;
   }
+
+  if (localhost_only) {
+    status = DDS::PropertyQosPolicyHelper::add_property(
+    participant_qos.property,
+    "dds.transport.UDPv4.builtin.parent.allow_interfaces",
+    "127.0.0.1",
+    DDS::BOOLEAN_FALSE);
```

Comment:
@BMarchi nit: I think indentation is off here, what did `uncrustify` say?

Reply:
uncrustify is happy

Reply:
Sorry, it wasn't